src/ghdcbot/engine/snapshots.py

```
@@ -164,6 +165,7 @@ def _collect_snapshot_data(
     contribution_summaries: list[ContributionSummary] | None,
     run_id: str,
     generated_at: datetime,
+    include_raw_events: bool = False,
```

Ruff (0.15.7)

[warning] Boolean-typed positional argument in function definition

(FBT001)